packages/nox/test/tpetra/ME_Tpetra_1DFEM_def.hpp

```diff
@@ -325,13 +325,19 @@ evalModelImpl(const Thyra::ModelEvaluatorBase::InArgs<Scalar> &inArgs,
     typedef typename tpetra_vec::execution_space execution_space;
 
     // Create ghosted objects
+    // Should there be braces corresponding to this if-statement and the following lines of code (as in xPtr_ case below)?
```

Comment:
@rppawlo can you confirm if braces should wrap the code after the if-statement, similar to the `xPtr_` case below?

Reply:
@ndellingwood - the uPtr code should NOT be wrapped in braces. Each iteration we will need a new import on the solution vector (uPtr). The xPtr holds nodal coords and is not changing from iteration to iteration so only needs to be called once. We can probably remove the modify_host() calls as @mhoemmen suggested.

Reply:
@rppawlo thanks for clarifcation.